### ISSUE TITLE
TensorFlow 2.0.0 without AVX

### ISSUE THREAD

**Issue by NaniSk** [fo40225/tensorflow-windows-wheel#139]
Hello, 
In first part, thanks for this project :)
If I I understood well, TF 2.0.0 without AVX instruction is in folder "/master/2.0.0/py37/CPU/sse2/tensorflow-2.0.0-cp37-cp37m-win_amd64.whl"?
If yes, I have another problem.
I tried install it with "pip install tensorflow-2.0.0-cp37-cp37m-win_amd64.whl" from download folder, but I have still have error: "tensorflow-2.0.0-cp37-cp37m-win_amd64.whl is not supported wheel on this platform".
Can you please help me?

Thanks

**Comment by fo40225** [comment]
@NaniSk 

Please don't change the filename and check your python is 64-bit or not.

**Comment by NaniSk** [comment]
Hi, thanks for fast answer. Im trying build it via Docker.
When I tried this via Python, I didnt have problem.
Did you try this in Docker too? :)

**Comment by fo40225** [comment]
If you are using linux container on windows, it is linux environment rather than windows.

**Comment by NaniSk** [comment]
Hi, thanks :) Do you have repo or can you recommend me some TF Linux without AVX?

**Comment by fo40225** [comment]
https://github.com/yaroslavvb/tensorflow-community-wheels/issues
https://github.com/lakshayg/tensorflow-build

**Comment by pratik-99** [comment]
hey I'm trying to download tensorflow 2 using >pip install tensorflow-2.0.0-cp37-cp37m-win_amd64.whl but it shows it is not a supported wheel on this platform I'm using 32 bit python can u recommend me which file should I download to install tensorflow 2 thank you in advance

**Comment by fo40225** [comment]
@pratik-99 https://github.com/fo40225/tensorflow-windows-wheel/tree/master/1.10.0/py36/CPU

**Comment by pratik-99** [comment]
@fo40225 I feel this is tensorflow 1.0 version, but the problem I am facing while loading my model on local system is that I am getting unknown layer: name error which I feel is because I am not using tensorflow 2 can you help me in any so that I could run tensorflow 2. I downloaded it but it gives me DLL import error

**Comment by fo40225** [comment]
Tensorflow 1.10 is my last build supports 32-bit python.

**Comment by pratik-99** [comment]
Yes, I upgraded to 64 bit python 3.7.9 and downloaded tensorflow 2.0 but it shows DLL import error

**Comment by fo40225** [comment]
Please check:
CPU has AVX support or not
install the newest msvc redist 2015-2019
exactly match python major version 3.X, use the newest minor version update
if you have nvidia GPU, use the exactly match cuda version(include major and minor version), exactly match the cudnn major version and the same or newer minor version

You should post the full error message.
